### DISCUSSION TITLE
appearence ?? fill

### DISCUSSION THREAD

**Discussion by iliassela** [myndpm/open-source#29] [Dynamic Forms]
i want to make appearence fill not outline how i can do it ?

**Comment by matheo** [comment]
Hi @iliassela!
That's controlled with the Angular Material configuration token in your module:
```
import { MAT_FORM_FIELD_DEFAULT_OPTIONS } from '@angular/material/form-field';

  providers: [
    {
      provide: MAT_FORM_FIELD_DEFAULT_OPTIONS,
      useValue: {
        appearance: 'fill',
        hideRequiredMarker: true,
        floatLabel: 'auto', // default value of INPUT.params.floatLabel
      },
    },
  ],
```
you can test it here: https://stackblitz.com/edit/myndpm-dyn-forms?file=src%2Fapp%2Fsimple-form%2Fsimple-form.module.ts

Perhaps I've used `outline` paddings in the default Dynamic Controls styles
but we could support the different themes with some additional work there

Looking forward for your impression and feedback ;)
